README.md

```diff
@@ -22,7 +22,7 @@ Create `.github/workflow/deploy.yaml` with the following to build on push.
 on:
   push:
     branches:
-      - "main"
+      - "master" # change to the branch you wish to deploy from
```

Comment:
The default branch should be main:
- https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches#about-the-default-branch
```suggestion
      - "main" # change to the branch you wish to deploy from
```